packages/medusa/src/services/order-edit-item-change.ts

```diff
@@ -68,6 +69,25 @@ export default class OrderEditItemChangeService extends TransactionBaseService {
     return itemChange
   }
 
+  async create(data: DeepPartial<OrderItemChange>): Promise<OrderItemChange> {
```

Comment:
suggestion: Not all the properties should be available, what about that (which is what I ve done in my pr):
```ts
export type CreateOrderEditItemChangeInput = {
  type: OrderEditItemChangeType
  order_edit_id: string
  original_line_item_id?: string
  line_item_id?: string
}
```